### PR TITLE
GDPR: show updated policy via modal

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

- displays modal via localStorage check + github login
- removed weird yellow hover on cross icon on modal
- update the background color on opened modal

@vs77bb @owocki @PixelantDesign 
On clicking **View Our Updated Terms** -> it takes you to `https://gitcoin.co/legal/policy `. 
I'm guessing this isn't, could get the right url ? 

@PixelantDesign could you give me the rgb color code of the blue background ? This is what I got , when I used a color picker -> It's weird :P 

screen shot 2018-05-24 at 4 13 02 pm

Or I can default it to the black background ^_^ 

screen shot 2018-05-24 at 4 13 18 pm

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
- css
- js

##### Fixes
 - #1239

### CODE REVIEW

**Comment by thelostone-mc** [comment]
Update: I think the privacy link is pending merge @ https://github.com/gitcoinco/web/pull/1222
Is that right @mbeacom  ? If yes, merge this post that

**Comment by SaptakS** [comment]
@thelostone-mc the link was already there. #1222 just updates and separates the html for terms and privacy.

File: app/assets/v2/css/base.css [hunk 1]
```diff
@@ -1671,3 +1676,20 @@ input.is-invalid {
...
+  letter-spacing: 0;
+}
+
+.modal.notify_policy_update {
+  max-width: 34rem;
+}
+
+.notify_policy_update .button {
+  padding: 4px 20px;
+}
```
Comment (SaptakS) [inline app/assets/v2/css/base.css]: Can you add a newline at the end? :sweat_smile:

**Comment by thelostone-mc** [comment]
@PixelantDesign @vs77bb can I just redirect it to https://gitcoin.co/legal/policy ?

File: app/assets/v2/css/base.css (hunk 1 above)
Comment (thelostone-mc) [reply app/assets/v2/css/base.css]: As in below the button ?

**Pull request merged**